app/code/community/Bolt/Boltpay/Helper/FeatureSwitch/ManagerTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+
+/**
+ * Trait Bolt_Boltpay_Helper_FeatureSwitch_ManagerTrait
+ *
+ * Provides method to get feature switches from bolt and save them into DB
+ *
+ */
+trait Bolt_Boltpay_Helper_FeatureSwitch_ManagerTrait
+{
+    use Bolt_Boltpay_Helper_GraphQLTrait;
+    /**
+     * This method gets feature switches from Bolt and updates the local DB with
+     * the latest values. To be used in upgrade data and webhooks.
+     */
+    public function updateSwitchesFromBolt()
+    {
+        $switchesResponse = $this->getFeatureSwitches();
+        $switchesData = @$switchesResponse->data->plugin->features;
+
+        if (!is_array($switchesData) || count($switchesData) == 0) {
+            return;
+        }
+
+        $switches = array();
+        foreach ($switchesData as $switch) {
+            $switches[$switch->name] = (object)array(
+                'value' => $switch->value,
+                'defaultValue' => $switch->defaultValue,
+                'rolloutPercentage' => $switch->rolloutPercentage
+            );
+        }
+        Mage::getModel('core/config')->saveConfig('payment/boltpay/featureSwitches', json_encode($switches));
+        Mage::getModel('core/config')->cleanCache();
+    }
+}
```

Comment:
For me, this is not a Helper, it is a model, especially when we are using it to persist to the database. Also, there is no need to over-complicate things with a FeatureSwitch directory and several class.  Everything that we are going to need here can go into `Bolt_Boltpay_Model_FeatureSwitch`.  

@johnmccombs1 How do you feel about this?

Reply:
It makes sense Leon. @johnmccombs1 waiting for your approval before moving this to model

Reply:
@vitaliyreznikov You can probably move forward with this simple change request.